src/components/login/ProviderLink.tsx

```diff
@@ -11,15 +11,24 @@ interface Props {
 }
 
 function loginMethod(provider: LoginProvider) {
+  switch (provider.method) {
+    case provider.method:
+      return provider.name
+
+    default:
+      break;
+  }
+}
+
+function loginButton(provider: LoginProvider) {
+  const logoPath = "/img/login/"
```

Comment:
Instead of using this switch statement, I'd probably just use string interpolation to match some standard naming convention for these logos. That way any future provider additions or removals won't require updating the code, but only adding the image in the right place. Can ditch this whole function in that case.